CICD/github/build.sh

```
@@ -24,11 +24,11 @@ sdk version
 javac -version
 export GRADLE_OPTS="-XX:MaxMetaspaceSize=512m -Xmx2g"
 ./gradlew --version
-# Check if we are in a fork or not
-if [ "$GITHUB_REPOSITORY" = "$GITHUB_HEAD_REPO_FULL_NAME" ]; then
+# Check if we have access to sonar token (we are not in a fork for example)
+if [ -n $SONAR_TOKEN ]; then
```

Shellcheck (0.10.0)

[error] -n doesn't work with unquoted arguments. Quote or use [[ ]].

(SC2070)